magenta/interfaces/midi/magenta_midi.py

```diff
@@ -89,7 +89,9 @@
 tf.app.flags.DEFINE_integer(
     'temperature_control_number',
     None,
-    'The control change number to use for controlling softmax temperature.')
+    'The control change number to use for controlling softmax temperature.'
+    'This should be an integer between 0 and 127 corresponding to a softmax '
+    'temperature between 0.1 and 2.')
```

Comment:
My hope was that I could add that while the temperature_control_number is between 0 and 127, this value is mapped to between 0.1 and 2. 

I was trying to follow the convention of the file, which lists for each of the flags:

"The control change number to use for..." or "The control change number to use with..." 

What about:

The control change number to use for controlling softmax temperature. The value of this control change number will be used to set the temperature in a linear range between 0.1 and 2. 

I guess that it is clear once you trace how it is being used here: https://github.com/tensorflow/magenta/blob/824ae077abcecc58c568b1a770c6c915dbca616b/magenta/interfaces/midi/midi_interaction.py#L103